waffle-doppelganger/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@ethereum-waffle/doppelganger",
+  "description": "Mock smart contracts in a smart way.",
+  "version": "0.0.1",
+  "author": "Marek Kirejczyk <account@ethworks.io> (http://ethworks.io)",
+  "repository": "git@github.com:EthWorks/Waffle.git",
+  "private": false,
+  "license": "MIT",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "solidity",
+    "testing",
+    "javascript",
+    "typescript",
+    "library"
+  ],
+  "homepage": "https://github.com/EthWorks/Waffle",
```

Comment:
Let's use: https://getwaffle.io/ (would be also good to change in other files as a separate PR)